apps/emqx/src/emqx_limiter/src/emqx_limiter_schema.erl

```diff
@@ -76,12 +119,20 @@ minimum_period() ->
 ref(Field) -> hoconsc:ref(?MODULE, Field).
 
 to_rate(Str) ->
+    to_rate(Str, true, false).
+
+to_burst_rate(Str) ->
+    to_rate(Str, false, true).
+
+to_rate(Str, CanInfinity, CanZero) ->
     Tokens = [string:trim(T) || T <- string:tokens(Str, "/")],
     case Tokens of
-        ["infinity"] ->
+        ["infinity"] when CanInfinity ->
             {ok, infinity};
+        ["0", _] when CanZero ->
+            {ok, 0}; %% for burst
         [Quota, Interval] ->
-            {ok, Val} = to_quota(Quota),
+            {ok, Val} = to_capacity(Quota),
             case emqx_schema:to_duration_ms(Interval) of
```

Comment:
could handle the case `10/s`?  I have to specify something like `10/1s`

Reply:
hocon only supports 10/1s, I think it should be consistent

Reply:
@zmstone what do you think?

Reply:
make sense to allow `10/s` and `10/m` etc. in HOCON